pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -633,6 +633,7 @@ public static void addColumnMetadataInfo(PropertiesConfiguration properties, Str
     properties.setProperty(getKeyFor(column, TOTAL_DOCS), String.valueOf(totalDocs));
     DataType dataType = fieldSpec.getDataType();
     properties.setProperty(getKeyFor(column, DATA_TYPE), String.valueOf(dataType));
+    properties.setProperty(getKeyFor(column, STORAGE_FORMAT), V1Constants.Str.STORAGE_FORMAT_DEFAULT_VALUE);
```

Comment:
Let's not add the storage format for now. I don't have a clear picture on how this field is going to be used. When we want to support another storage format, we can add it then.

Reply:
Removed.